bids/reports/report.py

```diff
@@ -0,0 +1,115 @@
+"""Generate publication-quality data acquisition methods section from BIDS dataset.
+"""
+from __future__ import print_function
+import json
+from os.path import dirname
+from os.path import abspath
+from os.path import join as pathjoin
+from collections import Counter
+
+from bids.reports import utils
+
+
+class BIDSReport(object):
+    """
+    Generates publication-quality data acquisition methods section from BIDS
+    dataset.
+
+    Parameters
+    ----------
+    layout : :obj:`bids.grabbids.BIDSLayout`
+        Layout object for a BIDS dataset.
+    config : :obj:`str` or :obj:`dict`, optional
+        Configuration info for methods generation. Can be a path to a file
+        (str), a dictionary, or None. If None, loads and uses default
+        configuration information.
+    """
+    def __init__(self, layout, config=None):
+        self.layout = layout
+        if config is None:
+            root = dirname(abspath(__file__))
+            config = pathjoin(root, 'config', 'converters.json')
+            with open(config) as fobj:
+                config = json.load(fobj)
+
+        if not isinstance(config, dict):
```

Comment:
I think you're missing the case where `config` is not `None`, but a filename is passed... this will fail on the current check.